src/ffmpeg-util.jl

```diff
@@ -58,11 +62,16 @@ struct VideoStreamOptions
         if format in ("mp4", "webm")
             (compression === nothing) && (compression = 20)
         end
+        
+        if format == "gif"
+            (loop === nothing) && (loop = 0)
```

Comment:
```suggestion
            isnothing(loop) && (loop = 0)

Reply:
I was trying to maintain consistency with the earlier lines of code by using `===` instead of `isnothing`, but if you prefer it this way, happy to switch!